webhooks-extension/README.md

```diff
@@ -52,20 +106,20 @@ Reference the [Knative eventing GitHub source sample](https://knative.dev/docs/e
 
 ## Limitations
 
-- Only Git Hub webhooks are currently supported.
-- Only `push` and `pull_request` events are supported at the moment. The webhook is currently only created with both `push` and `pull_request` events.
+- Only GitHub webhooks are currently supported.
+- Only `push` and `pull_request` events are currently supported, these are the events defined on the webhook.
 - All knative event sources are created in the namespace into which the dashboard and this extension are installed.
-- Only one webhook can be created for each git repository, so each repository will only be able to trigger a PipelineRun from one webhook.
+- Only one webhook can be created for each Git repository, so each repository will only be able to trigger a `PipelineRun` from one webhook.
 
-- Three pipeline definitions are currently supported.
+- The pipeline definitions that are currently supported are:
```

Comment:
Please either delete this or replace it with a definition of the conventions required by our current sink implementation.